src/components/quiz-list.tsx

```diff
@@ -134,22 +136,29 @@ export default function QuizList({
       {showSummary && (
         <div className="summary mb-6">
           <h3 className="text-xl font-semibold mb-2">Du hast es geschafft!</h3>
-          <p>
+          <p className="pb-6">
             Du hast{" "}
             {quizzesState
               .map((state) => state.isSolved)
               .reduce((acc, v) => acc + (v ? 1 : 0), 0)}{" "}
             von {quizzesState.length} Fragen richtig beantwortet.
           </p>
+          <Button onClick={() => router.push("/space/passwort/strength")}>
```

Comment:
Use the onFinish callback on the parent side. This way every quiz list would redirect to /space/passwort/strenth